addon/mixins/model-records/resource-route-mixin.js

```diff
@@ -2,40 +2,26 @@ import Ember from 'ember'
 import EmberDataRouteMixin from 'ember-data-route'
 
 const {
-  computed,
-  getOwner,
   Mixin,
-  String: { singularize }
+  inject: { service },
+  A
 } = Ember
 
-export default Mixin.create(EmberDataRouteMixin, {
-  templateAdminPath: null,
-
-  recordType: computed(function() {
-    return singularize(this.paramsFor('model-records').name)
-  }),
+const relationshipsFor = model => {
+  const relationships = []
 
-  templatePath: computed('recordType', function() {
-    return `${this.get('templateAdminPath')}/${this.get('recordType')}`
-  }),
+  model.get('constructor.relationshipsByName').forEach(relationship =>
+    relationships.push(relationship))
 
-  defaultTemplatePath: computed(function() {
-    return `${this.get('templateAdminPath')}/default`
-  }),
+  return A(relationships)
```

Comment:
I don't like that you had to do this to just get relationship names...

I started writing an alternative, but wound up with this, which isn't any better.

```js
const relationshipNamesByType = model.get('constructor.relationshipNames')
return Object.values(relationshipNamesByType).reduce((a, b) => a + b, A())
```

It blows my mind that `relationshipNames` returns an object with the relationships grouped by type, instead of...you know...the relationship names, like the name says.